packages/integration-tests/src/tests/api/profile/index.test.ts

```diff
@@ -122,6 +123,46 @@ describe('profile', () => {
       await deleteDefaultTenantUser(user.id);
       await deleteDefaultTenantUser(user2.id);
     });
+
+    it('should be able to update profile', async () => {
+      const { user, username, password } = await createDefaultTenantUserWithPassword();
+      const api = await signInAndGetUserApi(username, password);
+      const newProfile = {
+        profile: 'HI',
+        address: {
+          country: 'USA',
+        },
+      };
+
+      const response = await updateUser(api, { profile: newProfile });
+      // Should not be able to update address
+      expect(response).toMatchObject({
+        profile: {
+          profile: 'HI',
+        },
+      });
```

Comment:
Better add it as part of the test case description e.g `should not update profile address if the `Address` scope is missing.

Reply:
updated